site/docs/components/buttons.md

```diff
@@ -54,12 +72,30 @@ Secondary buttons are are White with Ignite Red text.
 
 ### Sizes
 
-Button sizes have dynamic widths based on a padding-left & padding-right of 24px plus the characters. Heights are set by adding add `.btn`, `.btn-lg` or `.btn-sm`.  Small buttons `.btn-sm` have a height of 32px. Base btns `.btn` have a height of 40px. Large buttons `.btn-lg` have a height of 52px.
+Heights are set by adding add `.btn`, `.btn-lg` or `.btn-sm`.  Small buttons `.btn-sm` have a height of 32px. Base btns `.btn` have a height of 40px. Large buttons `.btn-lg` have a height of 52px. Buttons have min-width of 154px, then have a dynamic width based on character counts and padding. Buttons widths can also be adjusted by adding a `.col-span-#` (# = 1-12) or by adding `.w-100`, to take up 100% of the parent element.
```

Comment:
Same, can remove the additional "add" - "Heights are set by adding **add"**

Reply:
@bexux Just pushed changes, let me know if anything else is needed. :)